frontend/benefit/applicant/src/components/applications/pageContent/usePageContent.ts

```diff
@@ -62,8 +67,20 @@ const usePageContent = (): ExtendedComponentProps => {
       existingApplicationStatus !== 'loading'
     ) {
       setIsLoading(false);
+      const existingApplicationCurrentStep = getApplicationStepFromString(
+        existingApplication?.application_step || DEFAULT_APPLICATION_STEP
+      );
+      if (applicationTempData.currentStep !== existingApplicationCurrentStep) {
```

Comment:
is if-check necessary? could `setCurrentStep(existingApplicationCurrentStep);` just work always?

Reply:
i ll recheck, but i think it creates infinite loop